src/cli.coffee

```diff
@@ -15,18 +9,25 @@ options = docopt """
   """
 
 
-generateConfig = ->
-  break for extension in extensions when options[extension]
+{docopt} = require 'docopt'
+options = docopt usage
+
+
+generateConfig = (extension) ->
+  asyncHandlers = require 'async-handlers'
+  fsExtra = require 'fs-extra'
```

Comment:
I was debating the same thing with myself  :)

Pros of moving requires up is that it makes code dry. Cons are that it makes refactoring harder, because once I use an external dependency a second time, I have to change the other method that uses it, even though this method isn't even related to my PR. 

Similarly, every time I remove a usage of an external dependency, I might have to move the require around to a more narrow scope. 

All of that sounds like a lot of unnecessary extra ceremony to me.